sdk/cosmos/src/operations/create_collection.rs

```diff
@@ -0,0 +1,101 @@
+use crate::headers::from_headers::*;
+use crate::prelude::*;
+use crate::resources::collection::{IndexingPolicy, PartitionKey};
+use azure_core::headers::{etag_from_headers, session_token_from_headers};
+use azure_core::{collect_pinned_stream, Request as HttpRequest, Response as HttpResponse};
+use chrono::{DateTime, Utc};
+
+#[derive(Debug, Clone)]
+pub struct Options {
```

Comment:
Understand regarding wanting the name to unambiguous when unqualified. 👍

However, I thought that the options were operation specific not client specific. In other words, creating a collection will take a different options type than deleting the collection. Is that not the case?

Reply:
Since the API is still in flux (old style vs. moving to Azure SDK guidelines) I wasn't sure, but if it's a per-method options, we *usually* follow <MethodName>Options so the options type is unambiguously tied to the actual method name. If it's shared by multiple methods, I would still qualify it but it could be something matching closer to the client.